frontend/src/lang/en.ts

```diff
@@ -2,9 +2,15 @@ const YES = "Yes";
 const NO = "No";
 const OTHER = "Other";
 const REFUSED = "Prefer not to answer";
-const UNKNOWN = "Unknown";
+export const UNKNOWN = "Unknown";
 const NOT_SURE = "Not sure";
 
+const FEMALE = "Female";
+const MALE = "Male";
+const TRANSWOMAN = "Trans femme or transwoman";
+const TRANSMAN = "Trans masculine or transman";
+const NONBINARY = "Nonbinary or gender non-conforming";
```

Comment:
Extending on [this comment](https://github.com/CDCgov/prime-simplereport/pull/7161/files#r1455657449), would love to couple these constants here with the value of the `GenderIdentities` dictionary we're defining so that there's one source of truth for these strings. Worry that we might have string drift if we redefine the constants here

Think we could replace these with `GenderIdentityDisplay.female`, `GenderIdentityDisplay.male`, etc.  so that the dictionary in the format file is the source of truth for what the strings are.